src/usr/local/www/diag_packet_capture.php

```diff
@@ -442,6 +443,12 @@ function fixup_host($value, $position) {
 		null,
 		'fa-stop-circle'
 	))->addClass('btn-warning');
+	if (file_exists($fp.$fns)) {
```

Comment:
There is still a problem here where immediately after clicking **Start**, this will display the wrong timestamp. It will either be missing if the file did not exist, or it will be the time of the previous capture, not the capture just started.

The `touch()` performed down lower in the diff needs to happen before this bit of code, not after. But the entire block of code below should not be moved, so perhaps a simple test before this point to check for the start action and run the `touch()` might work.

Reply:
fixed

tested with/without timestamp file, start/stop
now works correctly